src/reanimated2/PropAdapters.js

```diff
@@ -0,0 +1,46 @@
+import { createAnimatedPropAdapter } from './core';
+
+export const SVGAdapter = createAnimatedPropAdapter((props) => {
+  'worklet';
+  const keys = Object.keys(props);
+  // transform
+  if (keys.includes('transform')) {
+    if (Array.isArray(props.transform)) {
+      // case of array with 6 values => https://github.com/react-native-svg/react-native-svg/blob/develop/src/elements/Shape.tsx#L200
```

Comment:
This link can change, we should use links pinned to comment (clicking `y` key in GitHub UI changes link to commit one).

```suggestion
      // case of array with 6 values => https://github.com/react-native-svg/react-native-svg/blob/b2e2c355204ff4b10973d3afce1495f7e4167ff7/src/elements/Shape.tsx#L200
```

Reply:
good catch, THX!

Reply:
fixed e0bc1d5